pkgs/development/python-modules/earcut/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  yq-go,
```

Comment:
This argument is not needed

Reply:
Indeed, it's a leftover from the inline test that I temporarily included (see https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#writing-inline-package-tests).